lib/psci/psci_main.c

```diff
@@ -217,7 +217,7 @@ int psci_affinity_info(u_register_t target_affinity,
 
 	/* Calculate the cpu index of the target */
 	target_idx = plat_core_pos_by_mpidr(target_affinity);
-	if (target_idx == -1)
+	if (target_idx == -1U)
```

Comment:
Already addressed by #953 (by making target_idx signed, which is a better fix I think).